
int32 n_centroids

float32[] centroids_x
float32[] centroids_y
float32[] centroids_z
